modules/database/sqlite/can-load.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Can load function to determine if SQLite can be activated.
+ *
+ * @since n.e.x.t
+ * @package performance-lab
+ */
+
+/**
+ * Checks whether the given module can be activated.
+ *
+ * @since n.e.x.t
+ */
+return function() {
+	return class_exists( 'SQLite3' ) && wp_is_writable( WP_CONTENT_DIR );
```

Comment:
Is a minimum SQLite version required? I've seen stuff as early as 3.7 in the wild.

Reply:
Not as far as I know...
There's a check to determine if the version is > 3.7.11 to change the `can_insert_multiple_rows` prop in the object: https://github.com/WordPress/performance/blob/4badb58bd774b6bbaca8518a45d342443c286e5c/modules/database/sqlite/wp-includes/sqlite/class-perflab-sqlite-pdo-engine.php#L311-L313 and that is then used to determine which version of the insert queries functions to use - so it's compatible with both. https://github.com/WordPress/performance/blob/4badb58bd774b6bbaca8518a45d342443c286e5c/modules/database/sqlite/wp-includes/sqlite/class-perflab-sqlite-pdo-engine.php#L426-L432

Reply:
OK.